packages/web-app-files/src/views/Personal.vue

```diff
@@ -198,16 +204,73 @@ export default {
   },
 
   methods: {
-    ...mapActions('Files', ['loadIndicators', 'loadPreview']),
+    ...mapActions('Files', ['loadIndicators', 'loadPreview'], 'showMessage'),
     ...mapMutations('Files', [
       'SELECT_RESOURCES',
       'SET_CURRENT_FOLDER',
       'LOAD_FILES',
       'CLEAR_CURRENT_FILES_LIST',
-      'UPDATE_CURRENT_PAGE'
+      'REMOVE_FILE_FROM_SEARCHED'
     ]),
     ...mapMutations(['SET_QUOTA']),
 
+    async fileDropped(fileIdTarget) {
+      const selected = this.selectedFiles
+      const targetInfo = this.activeFiles.find(e => e.id === fileIdTarget)
+      const targetSelected = selected.some(e => e.id === fileIdTarget)
+      if (targetSelected) return
+      if (targetInfo.type !== 'folder') return
+
+      this.loading = true
+      const targetPath = targetInfo.path + '/'
+      const errors = []
+      const itemsInTarget = await this.getFolderItems(targetPath)
+      for (let i = 0; i < selected.length; i++) {
```

Comment:
Could you refactor this to `selected.forEach`?

Reply:
I'm not sure. Foreach loops costs a little more performance than for. https://levelup.gitconnected.com/which-is-faster-for-for-of-foreach-loops-in-javascript-18dbd9ffbca9